### PR TITLE
Added LinearPolar function

### PR DESCRIPTION
**Pull request by dlion**

### CODE REVIEW

**Comment by deadprogram** [comment]
Hi @dlion small merge conflict perhaps due to your other PR? Can you please rebase against dev? Thank you.

**Comment by dlion** [comment]
@deadprogram done, I think now you can merge and close it 😄

File: ROADMAP.md [hunk 1]
```diff
@@ -57,8 +57,8 @@ Your pull requests will be greatly appreciated!
         - [ ] [getRectSubPix](https://docs.opencv.org/master/da/d54/group__imgproc__transform.html#ga77576d06075c1a4b6ba1a608850cd614)
         - [ ] [initUndistortRectifyMap](https://docs.opencv.org/master/da/d54/group__imgproc__transform.html#ga7dfb72c9cf9780a347fbe3d1c47e5d5a)
         - [ ] [initWideAngleProjMap](https://docs.opencv.org/master/da/d54/group__imgproc__transform.html#gaceb049ec48898d1dadd5b50c604429c8)
-        - [X] [invertAffineTransform](https://docs.opencv.org/master/da/d54/group__imgproc__transform.html#ga57d3505a878a7e1a636645727ca08f51)
```
Comment (deadprogram) [inline ROADMAP.md]: You probably did not mean to unmark this line, since you did indeed implement this in a previously merged PR. ;)

**Comment by dlion** [comment]
@deadprogram you are totally right! :) Fix it!

**Comment by deadprogram** [comment]
Thanks for this contribution @dlion now squash/merging.

**Pull request merged**